config/config.d/test-config.yml

```diff
@@ -0,0 +1,8 @@
+version: 3
```

Comment:
what is this used for?

Reply:
that's the config file read from `config.d`

Reply:
is it used for testing?

Reply:
oh nm I see. Can we have these as strings in code that we write to the temp directory here: https://github.com/hootsuite/vault-ctrl-tool/blob/ffabbe0ce621e0e5fefc79f93ff4f8bc0f67758d/config/config_test.go#L104